data/human/free worlds 0 prologue.txt

```diff
@@ -1241,256 +1336,350 @@ mission "FW Katya 7C"
+			
+			`	"Yes, this a Free planet, but this evidence was far too blatant," Eyes responds. "Just because we found a match here doesn't mean this is where all the testing happened. I'm only detecting enough isotopes to account for one explosion, and you wouldn't just test once. If anything, this shows that the Free Worlds is being framed by someone, and they used one nuke here before bombing Geminus and Martini to make it look like it was the Free Worlds who did it."`
+			`	"I swear before God when I say that the Free Worlds has nothing to do with this, Captain," Katya says. "But we'll find whoever is responsible and bring them to justice."`
 			
+			label now
+			`	Katya thinks for a moment. "Captain, is the outpost reporting any job openings?"`
 			`	You check your computer. "One job opening, for a janitor in the mess hall."`
 			`	She grimaces. "Okay, it's better than nothing. I'll take the job to go undercover and see what I can find out. You take Eyes to <planet>, where he can get in touch with the Free Worlds Council. I'll find a way to send you a message once I've found the perpetrators."`
-			`	"Katerina," says Eyes, "this is way too dangerous."`
+			`	"Katerina," says Eyes, "this is way too dangerous. As I said, if we were able to detect this then so can the Navy, and they probably already have. They could decide to occupy Clink at any moment, and you'd be trapped here with no way out."`
 			`	"There's no other way," she says. "We have no time to lose. I'm willing to risk it."`
-			`	With deep reservations, you drop her off at the outpost, where she is quickly offered the job. "Not many folks care to work up here," says the foreman, "so your help would be much appreciated." You and Eyes return to your ship and get ready to travel to <planet> with the evidence.`
+			`	With deep reservations, you drop her off at the outpost. Katya ruffles her hair and rubs some moon dust onto her face so that she isn't recognized and is quickly offered the job after asking. "Not many folks care to work up here," says the foreman, "so your help would be much appreciated." You and Eyes return to your ship and get ready to travel to <planet> with the evidence.`
 				accept
 	
 	on visit
 		dialog `You land on <planet>, but you realize that Mr. Eyes is on one of your escorts that hasn't entered the system yet. Better depart and wait for it to arrive.`
 	on complete
-		"assisted free worlds" ++
 		payment 250000
+		"assisted free worlds" ++
 		conversation
-			`Eyes has been really quiet for the whole journey; you're sure that he's worried about Katya. When you land, he hands you <payment> and says, "Thank you for all your help, Captain. I hope that this leads to justice being done."`
+			`Eyes has been really quiet for the whole journey; you're sure that he's worried about Katya. When you land, he hands you <payment> and says, "Thank you for all your help, Captain. I pray that this leads to justice being done."`
```

Comment:
People aren't defined by the circumstances of their birth. Smh.

Reply:
No, but if we're talking regional cultures influencing language choice, unless he's spent a good decade specifically in the dirt belt or the south where people are more religious, his language is probably drawn from where and how he was raised.

Reply:
> Your justification for this whole thing seems to be what MZ said about a person from the Dirt Belt probably being religious.

That wiki page mentioning the Dirt Belt as being religious really has nothing to do with it.

> Alondo's sexuality is a remnant of my original plan for each of the three factions to be diverse in some ways but closed-minded in others: the Free Worlds being tolerant in that respect but *pretty homogeneous* racially and *religiously*, for example. Those themes don't yet get fleshed out much in the actual game, though. [cite](https://github.com/endless-sky/endless-sky/pull/3705#issuecomment-385248733)

If something is homogenous then it is largely made up of a single thing, and I think it'd be fair to say that the FW isn't homogenously atheistic. Part of this PR is bringing out themes that weren't too well fleshed out originally, with this being one of them. It's not as if there was absolutely nothing in the way of the Free Worlds being religious and I'm now adding a bunch of stuff in just because of one line from MZ, these themes already existed somewhat (see: Tomek having the Navy swear before God, Harmony and New Tibet both being religious worlds that end up within the FW), MZ stated that these themes weren't too well fleshed out despite his original plans, and I'm going back through and fleshing them out a bit more.

In the specific case of Ijs, he spent some time living on Harmony with the monks in his past. He's also dating Katya, who grew up in the South, and one's significant other certainly has a great influence on them. He also states when you meet him that he has many friends within the Free Worlds, so his friend group would include religious people, and I don't think it's a stretch to say that our friend groups have a great influence on our language in addition to our cultural background. These are the factors that led me to making the change in this specific case. (Note that me changing "hope" -> "pray" where both words could be used wasn't just a blanket replacement; where are some cases where I intentionally left it as "hope.") But Ijs is certainly the biggest outsider when it comes to the main cast of FW-affiliated characters, so I can revert this one.